lib/pages/home/home_page.dart

```diff
@@ -29,9 +30,14 @@ class HomePage extends StatelessWidget {
               User user = snap.data;
               firstName = user.firstName;
             }
-            return Text(firstName == null
-                ? S.of(context).messageWelcomeSimple
-                : S.of(context).messageWelcomeName(firstName));
+            return FlatButton(
```

Comment:
Add a better preview of the FAQ page on the home page: https://github.com/acs-upb-mobile/acs-upb-mobile/commit/4d7ade4fd707301f4452b7dfc81777bc3a4b0669